### ISSUE TITLE
Change Prometheus scrape interval for nightly and weekly clusters.

### ISSUE THREAD

**Issue by dekiel** [kyma-project/test-infra#1724]
Kyma Prometheus is set to scrape metrics every 30 seconds. This is too often for Prometheus Stackdriver collector which accept 1 minut as the lowest value. With scrape every 30s colelctor is failing on sending timeseries metrics with following error message.

```
level=debug ts=2019-11-13T11:23:14.675Z caller=series_cache.go:361 component="Prometheus reader" msg="unknown resource" labels="{endpoint=\"http-metrics\", instance=\"10.0.0.23:10255\", job=\"kubelet\", namespace=\"kube-system\", node=\"gke-weekly-default-pool-339ca0dd-nnjz\", service=\"kubelet\"}" discovered_labels="{__address__=\"10.0.0.23:10255\", __meta_kubernetes_endpoint_address_target_kind=\"Node\", __meta_kubernetes_endpoint_address_target_name=\"gke-weekly-default-pool-339ca0dd-nnjz\", __meta_kubernetes_endpoint_port_name=\"http-metrics\", __meta_kubernetes_endpoint_port_protocol=\"TCP\", __meta_kubernetes_endpoint_ready=\"true\", __meta_kubernetes_endpoints_name=\"kubelet\", __meta_kubernetes_namespace=\"kube-system\", __meta_kubernetes_service_label_k8s_app=\"kubelet\", __meta_kubernetes_service_name=\"kubelet\", __metrics_path__=\"/metrics\", __scheme__=\"http\", _kubernetes_cluster_name=\"weekly\", _kubernetes_location=\"europe-west4\", _stackdriver_project_id=\"sap-kyma-prow-workloads\", job=\"kyma-system/monitoring-exporter-kubelets/0\"}"
level=debug ts=2019-11-13T11:23:27.676Z caller=series_cache.go:361 component="Prometheus reader" msg="unknown resource" labels="{endpoint=\"https\", instance=\"kubernetes.default.svc:443\", job=\"apiserver\", namespace=\"default\", service=\"kubernetes\"}" discovered_labels="{__address__=\"35.204.63.157:443\", __meta_kubernetes_endpoint_port_name=\"https\", __meta_kubernetes_endpoint_port_protocol=\"TCP\", __meta_kubernetes_endpoint_ready=\"true\", __meta_kubernetes_endpoints_name=\"kubernetes\", __meta_kubernetes_namespace=\"default\", __meta_kubernetes_service_label_component=\"apiserver\", __meta_kubernetes_service_label_provider=\"kubernetes\", __meta_kubernetes_service_name=\"kubernetes\", __metrics_path__=\"/metrics\", __scheme__=\"https\", _kubernetes_cluster_name=\"weekly\", _kubernetes_location=\"europe-west4\", _stackdriver_project_id=\"sap-kyma-prow-workloads\", job=\"kyma-system/monitoring-exporter-kubernetes/0\"}"
```
Changing scrap intervall has to be done before Kyma installation so metrics from installation can be delivered to google stackdriver monitoring.

**Comment by dekiel** [comment]
Log messages presence has to be reviewed again. All Kyma prometheus scrape configs are set with 30sec interval. I didn't saw such messages recently and it's occurence might be related to OOM reason for container restarts which happened in the past for collector container.

**Comment by adamwalach** [comment]
No longer needed
